main.scss

```diff
@@ -1,2 +0,0 @@
-@import "./src/email-only-signup";

```

Comment:
We need to retain the `main.scss` so the component works with Bower correctly. Usually `main.scss` looks a little more like this: https://github.com/Financial-Times/o-teaser/blob/master/main.scss where we include all dependencies and output the main Sass.

At some point it would also be good to make this component work with silent mode: http://origami.ft.com/docs/syntax/scss/#silent-styles see `o-teaser` here: https://github.com/Financial-Times/o-teaser/blob/master/main.scss#L26 but that would be a breaking change, so maybe just revert back to having the `main.scss` in this PR and we'll do a breaking release next time round to fix this.